.travis.yml

```diff
@@ -26,23 +25,23 @@ cache:
         - $HOME/.cache/pip
 
 before_script:
-    - "export DISPLAY=:99.0"
-    - "sh -e /etc/init.d/xvfb start"
-    - "sleep 3" # give xvfb some time to start
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+    - sleep 3 # give xvfb some time to start
 
 before_install:
-    - "wget https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-linux64.tar.gz"
-    - "mkdir geckodriver"
-    - "tar -xzf geckodriver-v0.15.0-linux64.tar.gz -C geckodriver"
-    - "export PATH=$PATH:$PWD/geckodriver"
-    - "pip install -U pip"
+    - wget https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-linux64.tar.gz
+    - mkdir geckodriver
+    - tar -xzf geckodriver-v0.15.0-linux64.tar.gz -C geckodriver
+    - export PATH=$PATH:$PWD/geckodriver
+    - pip install -U pip
 
 install:
-    - "pip install tox-travis"
+    - pip install tox-travis
+    - pip install -r requirements/tox.txt
```

Comment:
This doesn't need to be fixed right now, but the pip-tools PR should generate a `travis.txt` or such which includes `tox-travis`.